src/components/30-organisms/footer/ui.test.js

```diff
@@ -0,0 +1,236 @@
+  await t.resizeWindow(575, 400);
+  await t.expect($socialMediaTitle.getStyleProperty('display')).eql('none');
+  await t.resizeWindow(767, 400);
+  await t.expect($socialMediaTitle.getStyleProperty('display')).eql('none');
+  await t.resizeWindow(991, 400);
+  await t.expect($socialMediaTitle.getStyleProperty('display')).eql('none');
+  await t.resizeWindow(992, 400);
+  await t.expect($socialMediaTitle.getStyleProperty('display')).eql('block');
+  await t.resizeWindow(1200, 400);
+  await t.expect($socialMediaTitle.getStyleProperty('display')).eql('block');
+});
+
+fixture('Footer - React Smoketest').page(
+  `${host}/iframe.html?id=organisms-footer-react--feature-footer-callbacks`
+);
+
+test('should render footer with working react callbacks', async t => {
+  const $axaElem = await Selector(TAG);
+  await t.expect($axaElem.exists).ok();
+  const $axaElemShadow = await Selector(
+    () => document.querySelector('axa-footer').shadowRoot
+  );
+  const $axaElemShadowEl = await $axaElemShadow.find(CLASS);
+  await t.expect($axaElemShadowEl.exists).ok();
+
+  await FooterAccessor.assertBackgroundColor(t, $axaElemShadowEl);
+
+  const $contactLink = FooterAccessor.getSlotNode('column-0-item-0');
+
+  await t.expect($contactLink.textContent).eql('Contact');
+  await t.expect($contactLink.visible).ok();
+
+  const $result = Selector('#clicked-link');
+  await t.expect($result.innerText).contains(' -');
+
+  await t.click($contactLink);
+
+  await t
+    .expect($result.innerText)
+    .contains('https://axa.ch/en/private-customers.html');
+
+  const $axaWorldwideLink = FooterAccessor.getSlotNode(
+    'column-0-title-desktop'
+  );
+
+  await t.expect($axaWorldwideLink.visible).ok();
+  await t.expect($axaWorldwideLink.textContent).eql('axa & you');
+
+  const $facebookButton = FooterAccessor.getSlotNode('column-social-item-0');
+
+  await t.expect($facebookButton.visible).ok();
+  await t.click($facebookButton);
+
+  await t.expect($result.innerText).contains('https://www.facebook.com/axach/');
+});
+
+fixture('Footer - Demo Smoketest').page(
+  `${host}/iframe.html?id=organisms-footer-demos--feature-footer-callbacks`
+);
+
+test('should render footer with working native callbacks', async t => {
+  const $axaElem = await Selector(TAG);
+  await t.expect($axaElem.exists).ok();
+  const $axaElemShadow = await Selector(
+    () => document.querySelector('axa-footer').shadowRoot
+  );
+  const $axaElemShadowEl = await $axaElemShadow.find(CLASS);
+  await t.expect($axaElemShadowEl.exists).ok();
+
+  await FooterAccessor.assertBackgroundColor(t, $axaElemShadowEl);
+
+  const $contactLink = FooterAccessor.getSlotNode('column-0-item-0');
+
+  await t.expect($contactLink.textContent).eql('Contact');
+  await t.expect($contactLink.visible).ok();
+
+  const $result = Selector('#clicked-link');
+  await t.expect($result.innerText).contains(' -');
+
+  await t.click($contactLink);
+
+  // Travis cannot handle the following block of code, for whatever reason.
```

Comment:
I wouldn't leave large blocks of commented-out code in, that's what version control is there  for

Reply:
Right, usually that's my feedback. Should also apply to myself. :)